pkg/store/consul/rcstore/fake_store.go

```diff
@@ -62,6 +63,10 @@ func (s *fakeStore) Create(manifest manifest.Manifest, nodeSelector labels.Selec
 	return entry.RC, nil
 }
 
+func (s *fakeStore) CreateTxn(txn *transaction.Tx, manifest manifest.Manifest, nodeSelector labels.Selector, podLabels labels.Set) (fields.RC, error) {
+	panic("transactions not implemented in fake rc store")
```

Comment:
i should add a comment explaining this. The decision I've made is that if a test uses transactions, it will use a real consul store instead of one of these fake storage layers. It would be complicated to implement the same transaction semantics ourselves.

I'm not sure exactly what helper functions you're asking about but all of the tests in integration_test.go use transactions against a real consul instance (i.e. they don't use this fake store)